### PR TITLE
chore(package): Use npm instead of yarn

### PR DESCRIPTION
**Pull request by jhermsmeier**
No need for yarn here

Change-Type: patch

### CODE REVIEW

**Review by Shou — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,11 @@ All notable changes to this project will be documented in this file
 automatically by Versionist. DO NOT EDIT THIS FILE MANUALLY!
 This project adheres to [Semantic Versioning](http://semver.org/).
 
+## v1.1.3 - 2018-04-11
+
+* Chore(package): Use npm instead of yarn #128 [Jonas Hermsmeier]
+* Fix: Replace Gitter link with forums link #128 [Andrew Scheller]
```
Comment (lurch) [inline CHANGELOG.md:10]: That's weird - my fix was actually in #125

Comment (jhermsmeier) [reply CHANGELOG.md:10]: Yeah, but versionbot can't merge external contributions, so that one got batched

Comment (lurch) [reply CHANGELOG.md:10]: Yeah, it makes sense that versionbot batched my fix (along with yours) into `v1.1.3` but I don't think it makes sense that versionbot has claimed that both changes were made in PR #128 ?

Or is the PR number that VB adds into `CHANGELOG.md` (for each line) always the PR in which the version-bump was done, and not necessarily the PR in which each change was made? :confused:

Comment (jhermsmeier) [reply CHANGELOG.md:10]: Oh, I see what you mean now – yeah, that's probably a bug, since it doesn't really support batching PRs

Comment (lurch) [reply CHANGELOG.md:10]: ping @hedss :wink:

Comment (jhermsmeier) [reply CHANGELOG.md:10]: I've already put this PR into a list of VB related failures, no need to ping ;)

**Pull request merged**